yang-module-versioning/draft-ietf-netmod-yang-module-versioning.txt

```diff
@@ -1373,117 +1160,47 @@ Internet-Draft    Updated YANG Module Revision Handling     October 2023
           non-backwards-compatible revision extension statement";
      }
 
-     extension label {
```

Comment:
We still need this extension to tag the revision statements?

Reply:
I think that labels are moving out of the base module versioning draft into the Semver draft.